jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/impl/model/TaskEventImpl.java

```diff
@@ -75,6 +74,8 @@
   private String correlationKey;
 
   private Integer processType;
+  
+  private String actualOwner;
```

Comment:
I would argue about this. you can target a group or something like  and then the assignment strategy kicks in so the event and actual owner does not need to be the same.